### DISCUSSION TITLE
GitHub repository

### DISCUSSION THREAD

**Discussion by RavinaShende** [community/community#34098] [Repositories] [answered]
fatal: not a git repository (or any of the parent directories): .git 
how I overcome with this error

**Comment by SilasRodrigues19** [comment]
@RavinaShende you need to initialize the repo, with the following command:

```git
git init
```

**Comment by RaheemAmer** [comment]
The command has to be entered in the directory of the repository. The error is complaining that your current directory isn't a git repo

- Are you in the right directory? Does typing ls show the right files?
- Have you initialized the repository yet? Typed git init? ([git-init documentation](https://git-scm.com/docs/git-init))

Either of those would cause your error.

Source:
- https://stackoverflow.com/questions/20413459/fatal-not-a-git-repository-or-any-of-the-parent-directories-git

**Reply by jeaolu** [reply]
This worked for me

**Comment by acezxn** [comment, marked as answer]
The reason for this error is that the working directory is not a git repository. In other words, it does not contain ```.git/``` directory in it.
To check whether your repository is initialized, you can try to list all files (including the hidden files) and look for ```.git/```.

If you are in the right directory and it does not contain ```.git/```, please run ```git init```.

**Reply by RavinaShende** [reply]
Thanks for solving the error
